uppsrc/plugin/ppm/ppm.cpp

```diff
@@ -16,28 +16,59 @@ bool PPMRaster::Create()
 		SetError();
 		return false;
 	}
-
-	try {
-		if(stream.GetLine() != "P6")
-			return false;
-		String h = stream.GetLine();
-		CParser p(h);
-		size.cx = p.ReadInt();
-		size.cy = p.ReadInt();
-		if(size.cx <= 0 || size.cx > 99999 || size.cy <= 0 || size.cy >= 99999)
-			return false;
-		h = stream.GetLine();
-		CParser p1(h);
-		int maxval = p1.ReadInt();
-		if(maxval <= 0 || maxval > 65535)
-			return false;
-		is16 = maxval > 255;
-		pixel_pos = stream.GetPos();
-		return true;
+	
+	auto SkipWhitespaces = [&] {
+		for(;;) {
+			while(IsSpace(stream.Peek()))
+				(void) stream.Get();

```

Comment:
[[maybe_unused]] attribute is cleaner solution. https://en.cppreference.com/w/cpp/language/attributes/maybe_unused.html

Reply:
Done.